man/fi_efa.7.md

```diff
@@ -97,6 +97,11 @@ provider for AWS Neuron or Habana SynapseAI.
   the endpoint is enabled. Otherwise, the call will fail. Also note that this
   option only applies to RDM endpoint.
 
+*FI_OPT_EFA_EMULATED_READ, FI_OPT_EFA_EMULATED_WRITE, FI_OPT_EFA_EMULATED_ATOMICS - bool*
+: These options only apply to the fi_getopt() call.
+  They are used to query the EFA provider to determine if the endpoint is  emulating Read, Write, and Atomic operations (return value is true), or if
```

Comment:
Telling from the rest of the file, this line has probably exceeded the max length.

Reply:
Looks like a missed line ending and didn't notice because I keep my window narrow.  Thank you, fixed!